k8s/rke2-mc/terraform.auto.tfvars.template

```diff
@@ -0,0 +1,48 @@
+environment                        = ""
```

Comment:
there's no reference to this file in the readme.

and should contain mostly the mandatory variables